tedana/combine.py

```diff
@@ -107,8 +107,11 @@ def make_optcom(data, tes, mask, t2s=None, combmode='t2s', verbose=True):
         Concatenated BOLD data.
     tes : (E,) :obj:`numpy.ndarray`
         Array of TEs, in seconds.
-    mask : (S,) :obj:`numpy.ndarray`
-        Brain mask in 3D array.
+    adaptive_mask : (S,) :obj:`numpy.ndarray`
+        Adaptive mask of the data indicating those voxels for which at least
```

Comment:
Since the thresholding for >= 3 voxels occurs internally, I'd pare down this docstring:

```suggestion
                Adaptive mask of the data indicating the number of echos with signal at each voxel
```

And move the admonition re: later echos into the function description !